### PR TITLE
feat: [SRM-12799]: Move Code Error headers to MFE

### PR DESCRIPTION
**Pull request by David-Kreiner**
and modify routes to match new format on ET MFE

### Summary

Error Tracking MFE needed to move dependency of the header bar for our standalone experience from Harness Core UI to our own repository. In addition to those changes some small routing changes were necessary to accommodate routing changes in our MFE.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by David-Kreiner** [comment]
retrigger jest

**Comment by David-Kreiner** [comment]
retrigger cypress-rest

**Review by cseverino789 — APPROVED** [review]

**Review by angelo-m-rodriguez — DISMISSED** [review]

**Comment by David-Kreiner** [comment]
retrigger cypress-cv

**Comment by David-Kreiner** [comment]
retrigger coverage

**Review by cseverino789 — APPROVED** [review]

File: src/modules/85-cv/components/LogsAnalysis/components/LogAnalysisRow/LogAnalysisRow.utils.ts [hunk 1]
```diff
@@ -65,8 +65,15 @@ export const onClickErrorTrackingRow = (
...
   })
-  const baseUrl = window.location.href.split('#')[0]
-  window.open(`${baseUrl}#${errorTrackingBaseUrl}/arc?event=${btoa(arcJson)}`)
+  let baseUrl = ''
+
+  if (window.browserRouterEnabled) {
+    baseUrl = '/ng'
+  } else {
+    baseUrl = window.location.href.split('#')[0] + '#'
+  }
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/LogsAnalysis/components/LogAnalysisRow/LogAnalysisRow.utils.ts:74]:
```suggestion
  let  baseUrl = window.location.href.split('#')[0] + '#'
  if (window.browserRouterEnabled) {
    baseUrl = '/ng'
  }
```

**Review by deepesh-ui — APPROVED** [review]

**Review by cjlee01 — APPROVED** [review]

**Pull request merged**